### DISCUSSION TITLE
README refers to FAQ, but FAQ is empty.

### DISCUSSION THREAD

**Discussion by vi** [dathere/qsv#532] [Q&A] [answered]
> See FAQ for more details.

It links to Github Discussions, to "FAQ" category. But it is empty (sans this discussion).

> It also has numerous new features & 60 additional commands/subcommands/operations (for a total of 80). 

Where is the full list? README only lists 47 commands in the table.

**Comment by jqnatividad** [comment, marked as answer]
> > See FAQ for more details.
> 
> It links to Github Discussions, to "FAQ" category. But it is empty (sans this discussion).

The FAQ is not empty - the FAQ category in discussions currently has 12 entries...

https://github.com/jqnatividad/qsv/discussions/categories/faq

> 
> > It also has numerous new features & 60 additional commands/subcommands/operations (for a total of 80).
> 
> Where is the full list? README only lists 47 commands in the table.

There are actually 81 commands/subcommands/operations - 47 commands, 5 apply subcommands, and 29 apply operations.

**Reply by vi** [reply]
Indeed, now it works, including when I click the link in README. Was it a Github glitch?
